README.md

```diff
@@ -52,14 +52,14 @@ with firewalld.
 
     ip link add "$BRIDGE" type bridge stp_state 0
     ip link set "$BRIDGE" up
-    ip addr add $NET.1/24 dev "$BRIDGE"
-    echo "allow $BRIDGE" >/etc/qemu/bridge.conf
-    firewall-cmd --zone trusted --add-interface=br_test
+    ip addr add "$NET.1/24" dev "$BRIDGE"
+    echo "allow $BRIDGE" >>/etc/qemu/bridge.conf
```

Comment:
In the Redhat POC I cheat here. I simply add "allow all" to the bridge.conf file... or I leave it to the installer to manage their own QEMU configuration.